commands/build.go

```diff
@@ -111,6 +112,7 @@ func (o *buildOptions) toControllerOptions() (*controllerapi.BuildOptions, error
 	}
 
 	opts := controllerapi.BuildOptions{
+		Ref:            identity.NewID(),
```

Comment:
This doesn't seem to be needed as already handled in `build/build` or `cmd/bake`